cmd/nitrite/main.go

```diff
@@ -23,17 +22,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	document, err := base64.StdEncoding.DecodeString(*fDocument)
```

Comment:
An attestation contains a `Document`, so calling the whole thing a document is a bit misleading